xero_accounting.yaml

```diff
@@ -27994,10 +27994,10 @@ components:
       - IGDSINPUT3
       - SROVR
       - TOURISTREFUND
-      - TXRCN33INPUT
-      - TXRCREINPUT
-      - TXRCESSINPUT
-      - TXRCTSINPUT
+      - TXRCN33
+      - TXRCRE
+      - TXRCESS
+      - TXRCTS
```

Comment:
@DanielGazzardXero to confirm, you're wanting to replace
```
- TXRCN33INPUT
- TXRCREINPUT
- TXRCESSINPUT
- TXRCTSINPUT
```

with

```
- TXRCN33
- TXRCRE
- TXRCESS
- TXRCTS
```

Reply:
Yes that is correct

Reply:
Yup, can confirm what Glen said. When adding the new 23 tax rates we took the opportunity to fix the existing rates which are currently wrongly affixed with input to the end.